internal/handlers/telegram/telegram.go

```diff
@@ -32,7 +32,14 @@ SendMessage sends a message to the Telegram channel specified in the settings
 func (tg *Client) SendMessage(msg string) {
 	newMsg := tgbotapi.NewMessage(tg.Settings.ChatID, "")
 	newMsg.Text = msg
-	tg.api.Send(newMsg)
+
+	if _, err := tg.api.Send(newMsg); err != nil {
+		// Keep trying until the message is successfully sent
+		for err != nil {
+			tg.logger.LogError(err)
+			_, err = tg.api.Send(newMsg)
+		}
```

Comment:
This loop forces to keep sending as long as the message did not go through. I thought about setting a number of trials before moving on, and would suggest ***resending 3 times and move on if it still does not go through.***

Reply:
@10eMyrT I think three attempts is fair. As long as each failed attempt is logged, which it looks like it is to me.

Reply:
I do too. I have changed the code to consider resending 3 times. Let's wait for other inputs before we merge.